src/formatDate.js

```diff
@@ -50,7 +50,50 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const newSeparator = toFormat[toFormat.length - 1];
+  const oldSeparator = fromFormat[fromFormat.length - 1];
+  const newFormat = toFormat;
```

Comment:
The 'newFormat' variable is redundant and unnecessary. You are creating a new reference but not changing the original array 'toFormat'. You can use 'toFormat' directly in your code.